cmake/multi_image.cmake

```diff
@@ -139,6 +144,7 @@ function(add_child_image_from_source)
     EXTRA_KCONFIG_TARGETS
     NCS_TOOLCHAIN_VERSION
     PM_DOMAINS
+    CONFIG_PARTITION_MANAGER_ENABLED # TODO is this set in child image?
```

Comment:
As stated in the kconfig, I think we should pass this as an overlay config file.